src/content/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring.mdx

```diff
@@ -164,20 +127,10 @@ Our infrastructure integrations give you access to the metrics of many popular s
 * Our [on-host integrations](/docs/infrastructure/host-integrations/host-integrations-list), like NGINX, Cassandra, MySQL, and many more: these work in concert with our infrastructure agent.
 * Make your own integration with [New Relic Flex](/docs/introduction-new-relics-flex-integration).
 
-<img
-  title="NR_Cluster_Explorer.png"
-  alt="Understand your Kubernetes nodes and pods, and how the relate to each other. "
-  src={infrastructurek8ClusterExplorer}
-/>
-
-<figcaption>
-  Our <DoNotTranslate>**[Kubernetes cluster explorer](/docs/integrations/kubernetes-integration/cluster-explorer/kubernetes-cluster-explorer)**</DoNotTranslate> gives you a powerful and innovative solution to the challenges associated with running Kubernetes at a large scale.
-</figcaption>
+## View logs for your infrastructure and app data [#logs-context]
 
-## Explore your infrastructure data [#explore]
+Bring your logs and infrastructure data together to make troubleshooting easier and faster! With logs in context, you can see logs of your infrastructure data, such as Kubernetes clusters. You can add as many configuration files as you need; these files will serve as the sources that push your log metadata to our platform.
 
-To learn more about features in the UI, see [Introduction to the infrastructure UI](/docs/infrastructure/infrastructure-ui-pages/infra-ui-overview). 
+You can also see log messages related to your errors and traces directly in your app's or host's UI. No need to switch to another UI page in New Relic! For more information, see our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context) documentation.
```

Comment:
I think you should end the first sentence with a : instead of a . if you're going to go with this structure. Just a minor thing